docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -9,8 +9,7 @@
             [(filter)]="filter"
             [allowNull]="allowNull"
             [maxHeight]="maxHeight"
-            [placeholder]="placeholder"
-            aria-label="Filter input">
```

Comment:
I would say that the `aria-label` attribute should be forwarded to the `button` element of `ux-input-dropdown` along with `aria-labelledby`. Buttons can certainly have an aria-label.

Reply:
Is there a way a forward this without creating a new input? There is already an aria-label input but that is for the filter input element.

Unless I create a second input called `ariaLabelButton` and document it? What are you thoughts on this.

Reply:
In that case, you'll still need to fix this issue rather than remove the aria-label from the example (the issue still exists, you've just hidden it). It is certainly wrong that the aria-label is forwarded to the filter input, but we can't change that without breaking existing installations.